afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ModificationScript.java

```diff
@@ -31,4 +35,19 @@ public String getScriptLabel() {
     public ScriptType getScriptType() {
         return ScriptType.valueOf(info.getGenericMetadata().getString(SCRIPT_TYPE));
     }
+
+    @Override
+    public void writeScript(String content) {
```

Comment:
same method as the write method of Action Script ! I think that you have to move up that method to AbstractScript.
in that case you have to implements Storable interface in AbstractScript.